doc-content/enterprise-only/openshift/operator-deploy-central-proc.adoc

```diff
@@ -39,7 +43,11 @@ In an authoring environment, if you want {CENTRAL} to push a project into an ext
 If you configure an external Maven repository (`MAVEN_REPO_URL`), change `MAVEN_MIRROR_OF` to exclude the artifacts in this repository from the mirror, for example, `external:*,!repo-custom`. Replace `repo-custom` with the ID that you configured in `MAVEN_REPO_ID`.
```

Comment:
Minor rewrite for "for example"

If you configure an external Maven repository (`MAVEN_REPO_URL`), change `MAVEN_MIRROR_OF` to exclude the artifacts in this repository from the mirror. For example, `external:*,!repo-custom`. Replace `repo-custom` with the ID that you configured in `MAVEN_REPO_ID`.

Reply:
Suggest to search other instances.